setup.ts

```diff
@@ -247,15 +247,11 @@ async function recaptcha(): Promise<void> {
   }
 }
 async function recaptchaSiteKey(): Promise<void> {
-  console.log(
-    "\nPlease visit this URL to set up reCAPTCHA:\n\nhttps://www.google.com/recaptcha/admin/create"
-  );
-  console.log(
-    '\nSelect reCAPTCHA v2 and the "I`m not a robot" checkbox option'
-  );
-  console.log(
-    '\nAdd "localhost" in domains and accept the terms, then press submit'
-  );
+  if (process.env.RECAPTCHA_SECRET_KEY) {
+    console.log(
+      ` \nreCAPTCHA secret key already exists with the value ${process.env.RECAPTCHA_SECRET_KEY}`
+    );
+  }
```

Comment:
If you enter a new secret, the old one still gets echoed to the screen

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/541d6adb-80a5-491c-b0f0-e1d7fa95a0f4)